include/callbacks.hh

```diff
@@ -37,45 +12,71 @@ int main () {
 #include <thread>
 #include <vector>
 
+/**
+ * alsa callback namespace
+ */
 namespace alsa_callback {
 /**
  * Alsa Callback Class. This reads Audio from the input hardware device and, using a callback will write these
  * to the correct pipes.
+ *
+ *  Example usage of this functionality:
+ *
+ * ===================================================
```

Comment:
This line of equals puts a horizontal divider in the docs website which doens't look great, can you remove it